stock_account_operating_unit/model/stock_quant.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# © 2015-17 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# © 2015-17 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from openerp import _, api, exceptions, models
+
+
+class StockQuant(models.Model):
+    _inherit = 'stock.quant'
+
+    @api.multi
+    def _account_entry_move(self, move):
+        """
+        Generate an accounting moves if the product being moved is subject
+        to real_time valuation tracking,
+        and the source or destination location are
+        a transit location or is outside of the company or the source or
+        destination locations belong to different operating units.
+        """
+        res = super(StockQuant, self)._account_entry_move(move)
+        if move.product_id.valuation == 'real_time':
+            # Inter-operating unit moves do not accept to
+            # from/to non-internal location
+            if (
+                move.location_id.company_id ==
+                    move.location_dest_id.company_id and
+                    move.operating_unit_id != move.operating_unit_dest_id
+            ):
+                err = False
+                if (move.location_id.usage != 'internal' and
+                        move.location_dest_id.usage == 'internal'):
+                    err = True
+                if (move.location_id.usage == 'internal' and
+                        move.location_dest_id.usage != 'internal'):
+                    err = True
+                if (move.location_id.usage != 'internal' and
+                        move.location_dest_id.usage != 'internal'):
+                    err = True
+                if err:
+                    raise exceptions.UserError(
+                        _('Transfers between locations of different operating '
+                          'unit locations is only allowed when both source '
+                          'and destination locations are internal.')
+                    )
+                src_company_ctx = dict(
+                    force_company=move.location_id.company_id.id
+                )
+                company_ctx = dict(company_id=move.company_id.id)
+                self = self.with_context(src_company_ctx)
+                data = move._get_accounting_data_for_valuation()
+                (journal_id, acc_src, acc_dest, acc_valuation) = data
```

Comment:
`(journal_id, acc_src, acc_dest, acc_valuation) = move._get_accounting_data_for_valuation()`? So you don't use the intermediate variable `data`?